os/include/sys/syscall.h

```diff
@@ -484,11 +484,12 @@
 #define SYS_recvfrom                   (__SYS_network + 8)
 #define SYS_recvmsg                    (__SYS_network + 9)
 #define SYS_send                       (__SYS_network + 10)
-#define SYS_sendto                     (__SYS_network + 11)
-#define SYS_setsockopt                 (__SYS_network + 12)
-#define SYS_shutdown                   (__SYS_network + 13)
-#define SYS_socket                     (__SYS_network + 14)
-#define __SYS_prctl                    (__SYS_network + 15)
+#define SYS_sendmsg                    (__SYS_network + 11)
```

Comment:
If you can add this new entry after SYS_eocket, then a single line change would be sufficient. Any particular reason for adding it in the middle of the list?

Reply:
We can also add it in this order for grouping same kind of syscall together.
```
SYS_recv
SYS_recvfrom
SYS_recvmsg
SYS_send
SYS_sendto
SYS_sendmsg
```
?

Reply:
My reasoning was to group all send syscall together in lexical order